### PR TITLE
Subgraph docs

### PR DESCRIPTION
**Pull request by shivgupt**

### CODE REVIEW

File: docs/stack/subgraph/queries.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
+You can use GraphQL queries to get quick info from [DAOstack Sunbgraph](https://thegraph.com/explorer/subgraph/daostack/v30_1) hosted on GraphExplorer.
```
Comment (ben-kaufman) [inline docs/stack/subgraph/queries.md]: Latest now is https://thegraph.com/explorer/subgraph/daostack/v31_0 but you might want to link here to master (https://thegraph.com/explorer/subgraph/daostack/master), we don't use it in production but it's updated to the latest master branch on the repo itself.

File: docs/stack/subgraph/queries.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+
+This guide explains how to use GraphQL queries to get single or multiple Entities and sort, filter and paginate them.
+
+The full list of `Subgraph Entities` cached by DAOstack subgraph can be found [here](../Entity)
+
+## General Guidelines
+
+  1. All queries must be wrapped inside `query {}` object.
+  2. Entity name is same as provided in [Entity list](../Entity) but in `lower case`.
```
Comment (ben-kaufman) [inline docs/stack/subgraph/queries.md]: starts with lower case letter (but camel case in general so should be `lowerCase` or so)

File: docs/stack/subgraph/queries.md [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+  3. You can [query for single entity](#query-for-single-entity) by providing Entity `id`.
+  4. You can [query for multiple Entities](#query-for-multiple-entities) by changing entity to plural. i.e. `proposal` -> `proposals`   
+  5. The `complex-field` i.e. fields that are themselves an Entity such as *dao* and *proposals* in above example, need to be wrapped in `{}` and provided with the subfield needed to be queried
```
Comment (ben-kaufman) [inline docs/stack/subgraph/queries.md]: Suggestion: `need to be wrapped in {}` -> need to be proceeded with `{}`

File: docs/stack/subgraph/queries.md [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+  6. Filter and Sort is not available for complex fields
```
Comment (ben-kaufman) [inline docs/stack/subgraph/queries.md]: This is not correct anymore. It used to not be available but it now is. See query like this for example:
```
{
  daos {
    proposals(first: 2) {
      id
    }
  }
}
```

File: docs/stack/subgraph/queries.md [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+  ) {
+    name
+    numberOfBoostedProposals
+  }
+}
+```
+  </body>
+</details>
+
+### Paginate
```
Comment (ben-kaufman) [inline docs/stack/subgraph/queries.md]: It's worth mentioning here that there is a 1000 entities limit per query.

File: docs/stack/subgraph/subgraphIntro.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
...
+**[Subgraph](https://github.com/daostack/subgraph) indexes the blockchain data and stores it in postgres database for easy and quick access.** The subgraph runs on a Graph Node which is a server that developers can run local or remote. The data store can be queried by GraphQL endpoints. [DAOstack subgraph](https://subgraph.daostack.io/) is based on graphprotocol, checkout [TheGraph](https://thegraph.com/docs/quick-start) for more details. TheGraph opens their server to others and you can find [daostack subgraph](https://thegraph.com/explorer/subgraph/daostack/v30_1) at Graph Explorer.
```
Comment (ben-kaufman) [inline docs/stack/subgraph/subgraphIntro.md]: Same as in the other file re the v30_1 URL

File: docs/stack/subgraph/queries.md (hunk 4 above)
Comment (shivgupt) [reply docs/stack/subgraph/queries.md]: Wow this is so awesome. Didn't know about it. Will add complex filter and sort examples here too

File: docs/stack/subgraph/queries.md (hunk 5 above)
Comment (shivgupt) [reply docs/stack/subgraph/queries.md]: Ohh yeah... I found that out just yesterday. 
Will add that too

File: docs/stack/subgraph/queries.md (hunk 4 above)
Comment (shivgupt) [reply docs/stack/subgraph/queries.md]: Ahh but we still can't filter based on complex fields.
Like if I want to get details of only those DAOs that has schemes' `version_gt: x`, I still can't do that in single query....can I?

Comment (ben-kaufman) [reply docs/stack/subgraph/queries.md]: Yes filtering based on such complex fields (basically filtering by fields of fields) is not supported.

**Pull request merged**